server/services/user.ts

```diff
@@ -86,3 +86,24 @@ export const makeUser = async ({ keycloakId, sites }) => {
     sites,
   });
 };
+
+export const getUserMigrations = async () => {
+  const users = await dbClient.db[collections.USER_MIGRATION]
+    .join({
+      userJoin: {
+        type: 'LEFT',
+        relation: collections.USERS,
+        on: {
+          'body.keycloakId': 'id::TEXT',
```

Comment:
Aren't we updating their keycloak id when we 'migrate' them right? So this won't join users that have already been updated, correct?

I'm okay with making this a "Users to be migrated" table though, and saying that its only for Pending users. We can remove the status column if we do end up doing that.